components/raftstore/src/store/fsm/peer.rs

```diff
@@ -4608,6 +4635,11 @@ where
         drop(meta);
         if is_leader {
             self.on_split_region_check_tick();
+        } else {
+            // Update the timestamp if there exists uncampaigned regions.
+            if !self.fsm.peer.uncampaigned_new_regions.0.is_empty() {
+                self.fsm.peer.uncampaigned_new_regions.1 = Instant::now();
```

Comment:
Why not update the timestamp when we push to the list (after line 4619)?

Reply:
It's redundant to record it with the same timestamp if the parent region spilts multi sub regions by `BatchSplit`.

As defined, the lifetime of the list `uncampaigned_new_regions` is kept until the current `Split` / `BatchSplit` is finished. Consequently, a single timestamp is enough to be utilized to mark its time-to-live (TTL).